### PR TITLE
Fix/data explorer route sync issues

### PR DESCRIPTION
**Pull request by connoratrug**
note: ran e2e locally ( passing) due to Jenkins issue 

#### Checklist
- [ ] Functionality works & meets specifications
- [ ] Code reviewed
- [ ] Code unit/integration/system tested
- [ ] User documentation updated
- [ ] Conventional commits (squash if needed)
- [ ] No warnings during install
- [ ] Updated javascript typing

### CODE REVIEW

**Comment by connoratrug** [comment]
[#7228 As explorer user I can choose what column to sort so I can more easily go through the data [2]](https://trello.com/c/Z0vveJBB/365-7228-as-explorer-user-i-can-choose-what-column-to-sort-so-i-can-more-easily-go-through-the-data-2)

File: packages/components-library/tests/unit/filters/containers/FilterContainer.spec.ts [hunk 1]
```diff
@@ -79,4 +79,9 @@ describe('FilterContainer.vue', () => {
     await wrapper.find('.btn.btn-block.btn-primary').trigger('click')
     expect(wrapper.find('.modal-dialog')).toBeDefined()
   })
+
+  it('reacts to filtersShown changes', async () => {
+    await wrapper.setProps({ filtersShown: ['color'] })
```
Comment (jelmerveen) [inline packages/components-library/tests/unit/filters/containers/FilterContainer.spec.ts:84]: better use includes, to make the test more robust?

File: packages/components-library/tests/unit/filters/containers/FilterContainer.spec.ts [hunk 2]
```diff
@@ -79,4 +79,9 @@ describe('FilterContainer.vue', () => {
...
+    expect((wrapper.vm).listOfVisibleFilters[0].label).toBe('Color')
```
Comment (jelmerveen) [inline packages/components-library/tests/unit/filters/containers/FilterContainer.spec.ts:85]: maybe map the array of objects to an array of labels and then use includes('Color') so if, for some instance the order changes the test won't break

**Review by jelmerveen — APPROVED** [review]
Approved, but maybe check my comment.

File: packages/components-library/src/components/ui/Pagination.vue [hunk 3]
```diff
@@ -178,18 +178,8 @@ export default {
...
-      // page, there are only 5 pages left. Make sure the currentPosition
-      // never exceeds the actual count.
-      if (currentPosition > this.localValue.count) {
-        currentPosition = this.localValue.count
-      }
-
-      const currentNewPage = Math.ceil(currentPosition / newSize)
-      this.updateValue({ page: currentNewPage })
+    'value.size' () {
+      this.updateValue({ page: 1 })
```
Comment (jvanveen) [inline packages/components-library/src/components/ui/Pagination.vue:182]: This only triggers a data update when a user is changing the page size from a page that is not 1.

**Review by jvanveen — CHANGES_REQUESTED** [review]

**Pull request merged**